ztp/gitops-subscriptions/argocd/PowerConfiguration.md

```diff
@@ -0,0 +1,224 @@
+
+```yaml
+workloadHints:
+  highPowerConsumption: false
+  perPodPowerManagement: false
+```
+
+If a cpu governor is configured via the additional kernel arguments, this must be removed according to the `complianceType` of the policy. If the `complianceType` is configured to `mustonlyhave`, then this can be accomplished by setting the `additionalKernelArgs` to the original list as defined in the `ztp/source-crs/PerformanceProfile.yaml` file. This is shown below:
+
+```yaml
+additionalKernelArgs:
+  - "cpufreq.default_governor=schedutil"
+  - "rcupdate.rcu_normal_after_boot=0"
+```
+
+If however the `complianceType` is configured to `musthave`, then the additional kernel argument must be removed as described in [NTO configuration hot fixes](https://github.com/openshift/cluster-node-tuning-operator/blob/master/docs/performanceprofile/configuration_hotfixes.md#additional-kernel-arguments).
+
+> **Note** that it is recommended to ensure that the `sysfs` plugin (responsible for capping the max cpu frequency) is not present in the TunedPerformancePatch profile.
+
+After the above policy changes have been made, it should be pushed to the Git repository and synced through argocd policy application. If the policy has a `remediationAction` set to `enforce`, then TALM (topology aware lifecycle manager) should do the needful of applying the policy to the effected SNOs. If however the `remediationAction` is configured for `inform`. Then a CGU (cluster group upgrade) CR must be created and applied to the hub cluster.
+
+After the policy has been applied to the effected SNOs, the change can be verified by viewing the PerformanceProfile on the SNOs:
+
+```sh
+oc get PerformanceProfile -o yaml --kubeconfig sno_kubeconfig.yaml
+```
+
+## Validating the Power configuration settings
+
+In order to validate whether the correct power configuration mode has been applied via ZTP post deployment, the `/proc/cmdline` config on the nodes can be verified as shown below.
+
+> **Note**: If power saving is enabled, `intel_pstate` is set to `passive`, otherwise for performance modes it is set to `disable`.
+
+```sh
+sh-4.4# cat /proc/cmdline
+BOOT_IMAGE=(hd0,gpt3)/ostree/rhcos-2d50e6ef6b1fc7512bb8021ef7c53860520d65fb2befead87504408cbc6b7488/vmlinuz-4.18.0-372.26.1.rt7.183.el8_6.x86_64 console=tty0 console=ttyS0,115200n8 ostree=/ostree/boot.0/rhcos/2d50e6ef6b1fc7512bb8021ef7c53860520d65fb2befead87504408cbc6b7488/0 ignition.platform.id=gcp root=UUID=7e3ae9c1-86f3-4d21-9c47-985394d42fe0 rw rootflags=prjquota boot=UUID=4adff52b-34e8-4fc2-9817-949108dbc896 skew_tick=1 nohz=on rcu_nocbs=1 tuned.non_isolcpus=00000001 systemd.cpu_affinity=0 intel_iommu=on iommu=pt isolcpus=managed_irq,1 nohz_full=1 tsc=nowatchdog nosoftlockup nmi_watchdog=0 mce=off skew_tick=1 rcutree.kthread_prio=11 default_hugepagesz=1G hugepagesz=2M hugepages=128 intel_pstate=passive cpufreq.default_governor=schedutil
```

Comment:
There are too many options here that could be different or change over time. I'd rather you just mention the options you expect to be there or not to be there.